experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/docker-compose.yaml

```diff
@@ -12,6 +12,6 @@ services:
     image: jenkins/ssh-agent:5.3.0-jdk11
     container_name: desktop-jenkins_agent-1
     environment:
-      - JENKINS_AGENT_SSH_PUBKEY=ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINVxUeiHKOsQBml4gDe4cAd8KGK4TZzYJGmqPDZgI6aE jh@jh-Modern-14-B5M
+      - JENKINS_AGENT_SSH_PUBKEY=ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIMYAhhPZeT0AiWIwfDggIexbgnPYX78Ehu9vL+/He0xG jh@jh-Modern-14-B5M
```

Comment:
Suggestion for the release candidate setup: have a setting in the compose file that will fail the `docker compose up` if the setup script has not been run before first execution.